src/scenes/Event/components/TransactionHistory/TxRow.js

```diff
@@ -52,14 +52,27 @@ export default class TxRow extends Component {
     this.setState({ expanded: !this.state.expanded });
   };
 
+  getAmount = (transaction) => {
+    const { txType } = transaction;
+    let { amount } = transaction;
+    if (txType === TransactionType.CREATE_EVENT) {
+      amount = transaction.escrowAmount;
+    } else if (txType === TransactionType.WITHDRAW) {
+      amount = stringToBN(transaction.winningAmount) + stringToBN(transaction.escrowWithdrawAmount);
```

Comment:
nit: handle `amount` parsing in the `Transaction` model. all models that are storing big numbers are converted to decimals. there are usually two fields, one for decimal and one for satoshi. see `Bet` model for reference.